### PR TITLE
Added PaperFamilyCount field

### PR DESCRIPTION
**Pull request by cyhuang01**

### CODE REVIEW

File: academic-services/graph/resources-faq.md [hunk 1]
```diff
@@ -59,4 +59,10 @@ It is relatively easy and inexpensive to compute citation counts and h-index usi
 
 ## What is FamilyId in Papers.txt
 
-Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value  being the PaperId of the main paper.
+Some papers were published in multiple venues, e.g. in archive and a conference. These papers share same FamilyId where the value being the PaperId of the main paper.
```
Comment (DarrinEide) [inline academic-services/graph/resources-faq.md:62]: How about this for a definition?
"A paper "family" is defined as a group of papers that have been determined to be the same fundamental paper, but published in different venues (e.g. pre-print and conference). In this case only one of the papers will be determined to be the "primary" paper, and each paper in the family group will have its "FamilyId" value set to the primary papers ID.

If the "FamilyId" value is not defined for a paper it means that the paper is not part of a larger family group, however it is still considered to be the primary paper of a single-paper family."

File: academic-services/graph/resources-faq.md [hunk 2]
```diff
@@ -59,4 +59,10 @@ It is relatively easy and inexpensive to compute citation counts and h-index usi
...
+
+## PaperCount vs. PaperFamilyCount
+
+PaperCount is the number of papers associated with the entity.
+
+PaperFamilyCount is similar as PaperCount where multiple papers with same FamilyId are only counted as one.
```
Comment (DarrinEide) [inline academic-services/graph/resources-faq.md:68]: >PaperFamilyCount is similar as PaperCount where multiple papers with same FamilyId are only counted as one [](start = 0, length = 106)

If we change the above definition, we could simply make this "PaperFamilyCount is the number of primary family papers associated with the entity."

File: academic-services/graph/reference-data-schema.md [hunk 3]
```diff
@@ -45,10 +45,11 @@ Column # | Name | Type | Note
 6 | OfficialPage | string |
 7 | WikiPage | string |
 8 | PaperCount | long |
-9 | CitationCount | long |
-10 | Latitude | float? |
-11 | Longitude | float? |
-12 | CreatedDate | DateTime |
+9 | PaperFamilyCount | long | See [FAQ](resources-faq.md#papercount-vs-paperfamilycount)
```
Comment (RickRogahn) [inline academic-services/graph/reference-data-schema.md:48]: Seems odd that this is the only attribute with an FAQ link, no?

Comment (cyhuang01) [reply academic-services/graph/reference-data-schema.md:48]: FamilyId field in Papers.txt also has a link to FAQ.

File: academic-services/graph/resources-faq.md (hunk 1 above)
Comment (cyhuang01) [reply academic-services/graph/resources-faq.md]: suggestion applied.

File: academic-services/graph/reference-data-schema.md (hunk 3 above)
Comment (DarrinEide) [reply academic-services/graph/reference-data-schema.md:48]: I agree with Rick, it feels like we should be a bit more diligent with our per-field descriptions

**Pull request merged**